cast/src/tx.rs

```diff
@@ -176,20 +183,34 @@ impl<'a, M: Middleware> TxBuilder<'a, M> {
                 self.chain.try_into().wrap_err("resolving via etherscan requires a known chain")?;
             get_func_etherscan(
                 sig,
-                self.to,
+                self.to.unwrap(),
```

Comment:
there might be circumstance where this is used outside of the cmd context so we should convert this into an error instead of unwrapping : `to.ok_or_else(|| eyre::eyre!("to must be set"))?`